packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -5619,51 +5610,60 @@ function mockListKVNamespacesRequest(...namespaces: KVNamespaceInfo[]) {
   );
 }
 
+interface ExpectedAsset {
+  filePath: string;
+  content: string;
+  expiration?: number;
+  expiration_ttl?: number;
+}
+interface UploadBody {
```

Comment:
should probably specify this is a static asset upload, because this test file has a bunch of other stuff going on